src/raster/r.damflood/main.c

```diff
@@ -872,21 +872,21 @@ int main(int argc, char *argv[]){
 		   shallow_water(m_h1,m_u1,m_v1,m_z,m_DAMBREAK,m_m,m_lake,m_h2,m_u2,m_v2,row,col,nrows,ncols,timestep,res_ew,res_ns,method,num_cell, num_break,t);
 		   
 
-    //*************************************** overwriting *********************************************
+    //*************************************** Overwriting *********************************************
     timestep_ct=0;
     if (t<TSTOP){   
-    /* open new cicle */
+    /* Open new cicle */
```

Comment:
What does "cicle" here mean? I'm sure it's from the Italian "ciclo"

Reply:
cycle